### ISSUE TITLE
Push the awesomeness to 11

### ISSUE THREAD

**Issue by ellisonbg** [cloudpipe/cloudpipe#40]
This is totally awesome, but I think it would be great if you could turn up the awesome all the way to 11. That way I could use it at my unicorn farm.

Can't wait to see this deployed on tmpnb 

;-)

**Comment by rgbkrk** [comment]
Are you saying we're making unicorns Brian?

**Comment by rgbkrk** [comment]
Also, it's semi-deployed right now on a subdomain of [cloudpi.pe](http://cloudpi.pe), feel free to reach me directly about trying it out.

**Comment by ellisonbg** [comment]
No, I have a unicorn farm and an awesomeness of 11 would increase yield and efficiency - there would be unicorns abundant

**Comment by rgbkrk** [comment]
Once we finish out v0.0.3, implementing multiple languages (including a forked Python3 client), I'd say this has been closed.

What do you think @ellisonbg?

**Comment by ellisonbg** [comment]
Yeah, I guess that is fine.

**Comment by sjobeek** [comment]
Excellent.  Way to be awesome. +11

I'd love to help test/debug when you're ready.
